### PR TITLE
add limit for mysql (slow when has large misfire job)

### PR DESCRIPTION
**Pull request by vinhhrv**
add limit for mysql (slow when has large misfire job)

### CODE REVIEW

File: src/Quartz/Impl/AdoJobStore/MySQLDelegate.cs [hunk 1]
```diff
@@ -34,5 +34,10 @@ protected override string GetSelectNextTriggerToAcquireSql(int maxCount)
         {
             return SqlSelectNextTriggerToAcquire + " LIMIT " + maxCount;
         }
+
+        protected override string GetSelectNextMisfiredTriggersInStateToAcquireSql(int maxCount)
+        {
+            return SqlSelectHasMisfiredTriggersInState + " LIMIT " + maxCount;
```
Comment (lahma) [inline src/Quartz/Impl/AdoJobStore/MySQLDelegate.cs]: HasMisfiredTriggersInState has API promise "The most misfired triggers to return, negative for all", how does MySQL limit behave with -1?

Comment (vinhhrv) [inline src/Quartz/Impl/AdoJobStore/MySQLDelegate.cs]: Sori lahma,

I miss this case,  i found this line
int maxMisfiresToHandleAtATime = (recovering) ? -1 : MaxMisfiresToHandleAtATime;
20 or -1

I just commited.

Tks.

**Comment by lahma** [comment]
Looking good, thanks for the contribution!

**Comment by vinhhrv** [comment]
Hi lahma,

Can you bump to new version for this issue ?,
I have waited for a long time

Thank you

**Pull request merged**